spec/cc/engine/gem_detector_spec.rb

```diff
@@ -0,0 +1,25 @@
+require "spec_helper"
+require "cc/engine/gem_detector"
+
+module CC::Engine
+  describe GemDetector do
+    describe "#has_gem?" do
+      before do
+        gemfile = File.read("Gemfile.lock")
```

Comment:
How about copying our current lockfile to a fixture so changes to our gems don't risk breaking this spec?